addon/components/base/bs-dropdown.js

```diff
@@ -154,7 +154,7 @@ export default Ember.Component.extend({
    * @private
    */
   containerClass: computed('toggle.tagName', 'direction', function() {
-    if (this.get('toggle.tagName') === 'button') {
+    if (this.get('toggle.tagName') === 'button' && this.get('toggle.block') !== true) {
```

Comment:
Either `!this.get('toggle.block')` or `this.get('toggle.block') === false`? The former is more permissive and would require the property to be initialized reliably.